### PR TITLE
Added Symfony command to check user status.

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
I needed a clean way to check user status anyway for a DMS ticket, and this may be handy to have available later, so adding command. Alternatively, we can also just keep it in the one-off's misc repo if preferred.

### CODE REVIEW

File: src/Command/CheckUserCommand.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+     * Symfony command execute section.
+     *
+     * @param InputInterface  $input  Required by Command.
+     * @param OutputInterface $output Required by Command.
+     *
+     * @throws \Exception If username not found.
+     *
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```
Comment (praneethpr) [inline src/Command/CheckUserCommand.php:69]: Needs to return 0 (for no issues) or 1 (if there are any). Most of the old commands return void, I don't know why we did that, but it looks like doesn't break anything though

https://symfony.com/doc/4.4/console.html#creating-a-command

File: src/Command/CheckUserCommand.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            $activeUser = false;
+            $person = $account->getPerson();
+            $researchGroups = $person->getResearchGroups();
+            foreach ($researchGroups as $researchGroup) {
+                if (false === $researchGroup->isLocked()) {
+                    $rgList[] = $researchGroup->getShortName();
+                    $activeUser = true;
+                }
+            }
+            if (true === $activeUser) {
```
Comment (praneethpr) [inline src/Command/CheckUserCommand.php:89]:
```suggestion
            if ($activeUser) {
```

File: src/Command/CheckUserCommand.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        $account = $accountRepository->findOneBy(
+            ['userId' => $username]
+        );
+
+        if ($account instanceof Account) {
+            $activeUser = false;
+            $person = $account->getPerson();
+            $researchGroups = $person->getResearchGroups();
+            foreach ($researchGroups as $researchGroup) {
+                if (false === $researchGroup->isLocked()) {
```
Comment (praneethpr) [inline src/Command/CheckUserCommand.php:84]:
```suggestion
                if (!$researchGroup->isLocked()) {
```

File: src/Command/CheckUserCommand.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+use App\Entity\Account;
+
+/**
+ * Command to convert a user into a Pelagos DRPM.
```
Comment (praneethpr) [inline src/Command/CheckUserCommand.php:16]: Change comment description

File: src/Command/CheckUserCommand.php [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+            ['userId' => $username]
+        );
+
+        if ($account instanceof Account) {
+            $activeUser = false;
+            $person = $account->getPerson();
+            $researchGroups = $person->getResearchGroups();
+            foreach ($researchGroups as $researchGroup) {
+                if (false === $researchGroup->isLocked()) {
+                    $rgList[] = $researchGroup->getShortName();
```
Comment (praneethpr) [inline src/Command/CheckUserCommand.php:85]: Can declare $rgList as empty array at the top

File: src/Command/CheckUserCommand.php (hunk 4 above)
Comment (fightingtexasaggie) [reply src/Command/CheckUserCommand.php:16]: oops, missed that. thx

File: src/Command/CheckUserCommand.php (hunk 1 above)
Comment (fightingtexasaggie) [reply src/Command/CheckUserCommand.php:69]: I'll do what Mike had started doing, with the @return int Return code.

**Review by praneethpr — APPROVED** [review]
LGTM!

**Pull request merged**